### PR TITLE
validation.md: Fix capitalization

### PR DESCRIPTION
**Pull request by mattayes**
Capitalize "it" to "It" at the start of a bullet point.

### CODE REVIEW

File: content/en/docs/usecases/validation.md [hunk 1]
```diff
@@ -78,7 +78,7 @@ Using CUE to generate the schema has several advantages over doing so directly:
 
 - CUE is far less verbose
```
Comment (myitcv) [inline content/en/docs/usecases/validation.md]: We could also add `.` to the end of the first two bullet points for completeness.

**Review by myitcv — APPROVED** [review]
Thanks for this. LGTM with one small suggestion.

File: content/en/docs/usecases/validation.md (hunk 1 above)
Comment (mattayes) [reply content/en/docs/usecases/validation.md]: @myitcv Can do 👍🏻

**Comment by myitcv** [comment]
Thanks for this, @mattayes

**Comment by mattayes** [comment]
Thanks for reviewing/merging @myitcv!

**Pull request merged**